notebooks/Liquidators_data.ipynb

```diff
@@ -636,14 +375,12 @@
     "common_loanid_df = common_loanid_df.drop(columns=['transaction_hash_repaid', 'transaction_hash_transferred'])\n",
     "# Display the resulting dataframe\n",
     "filtered_df_hstk_v1 = common_loanid_df\n",
-    "print(filtered_df_hstk_v1.columns)\n",
-    "print(list[filtered_df_hstk_v1[\"transaction_hash\"].shape])\n",
     "\n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 332,
```

Comment:
Can you pls add Nostra Alpha and Nostra Mainnet so that they appear in the charts, even if the data is empty? Or is it too difficult?